src/RazorSdk/Tasks/StaticWebAssets/GenerateStaticWebAssetsDevelopmentManifest.cs

```diff
@@ -8,6 +8,7 @@
 using System.Security.Cryptography;
 using System.Text.Encodings.Web;
 using System.Text.Json;
+using System.Text.Json.Serialization;
```

Comment:
Do you need this? It looks like the PR removed some of the JSON serialization, but did not add new ones.